subworkflows/local/pair_variant_calling.nf

```diff
@@ -98,26 +95,13 @@ workflow PAIR_VARIANT_CALLING {
         }
     }
 
-    if ('msisensorpro' in tools) {
-        MSISENSORPRO_MSI(
+    if (tools.contains('msisensorpro')) {
+        MSISENSORPRO_MSI_SOMATIC(
             cram_pair,
             msisensorpro_scan)
     }
 
-    if ('strelka' in tools)) {
-        STRELKA(
-            cram_pair,
-            fasta,
-            fasta_fai,
-            target_bed_gz_tbi)
-
-        strelka_indels_vcf = STRELKA.out.indels_vcf
-        strelka_snvs_vcf   = STRELKA.out.snvs_vcf
-
-        strelka_vcf = strelka_vcf.mix(strelka_indels_vcf,strelka_snvs_vcf)
-    }
-
-    if ('mutect2' in tools){
+    if (tools.contains('mutect2')){
```

Comment:
Shouldn't this part be removed?

Reply:
WIP to add Gavin's Tumor_somatic mutect2 workflow.

Reply:
but yes